### PR TITLE
LL-426 Improve queries through refs [NO MERGE]

### PR DESCRIPTION
**Pull request by ht2**

### CODE REVIEW

**Comment by ht2** [comment]
🌞

File: ~/tests/xapi-statements/storage/5988f0f00000000000000001/attachments/6e2008384385c9c216094f22d73347132cd84624167358972fed300f604770d2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+A
```
Comment (ryansmith94) [inline ~/tests/xapi-statements/storage/5988f0f00000000000000001/attachments/6e2008384385c9c216094f22d73347132cd84624167358972fed300f604770d2.txt]: Did you mean to commit this?

Comment (ht2) [reply ~/tests/xapi-statements/storage/5988f0f00000000000000001/attachments/6e2008384385c9c216094f22d73347132cd84624167358972fed300f604770d2.txt]: Looks like the storage folder wasn't as ignored as you would expect?

Comment (ht2) [reply ~/tests/xapi-statements/storage/5988f0f00000000000000001/attachments/6e2008384385c9c216094f22d73347132cd84624167358972fed300f604770d2.txt]: Or did I cancel a test... and then that change was there (which got added). I'll try and revert it

Comment (ht2) [reply ~/tests/xapi-statements/storage/5988f0f00000000000000001/attachments/6e2008384385c9c216094f22d73347132cd84624167358972fed300f604770d2.txt]: Removed. It had somehow symlinked my home directory into the project??!?!

**Pull request merged**